src/dataset-builder/DatasetBuilderUtils.ts

```diff
@@ -146,20 +146,20 @@ export const createSnapshotBuilderCountRequest = (cohort: Cohort[]): SnapshotBui
   return { cohorts: _.map(convertCohort, cohort) };
 };
 
-export const HighlightConceptName = ({ conceptName, searchFilter }): ReactElement => {
-  const startIndex = conceptName.toLowerCase().indexOf(searchFilter.toLowerCase());
+export const HighlightSearchText = ({ columnItem, searchFilter, style = {} }): ReactElement => {
+  const startIndex = columnItem.toLowerCase().indexOf(searchFilter.toLowerCase());
```

Comment:
So, I know this isn't related to your change, but something interesting here is that this will only bold the first instance of the searchTerm. If we wanted to bold all occurrences, we could try to do something recursive, where the last `span` is instead a HighlightSearchText component?

Reply:
Interesting idea - I'll look into this!

Reply:
Update: Implemented this ^